agile_problem3.exs

```diff
@@ -0,0 +1,25 @@
+defmodule Problem3 do
+	def check_duplicates(), do: -2
+	
+	def check_duplicates(list) do
+		#First check that we have at least 2 elements. Otherwise it's useless
+		if Enum.count(list)<2 do
+			-1
+		else
+			if list == Enum.uniq(list) do

```

Comment:
Nice try but this is cheating, `Enum.uniq` uses [maps in it's implementation](https://github.com/elixir-lang/elixir/blob/v1.5.0/lib/elixir/lib/enum.ex#L2594), you where not suppose to use any extra data structures.

The same comment of using numbers vs atoms apply.